examples/using-type-definitions/src/components/layout.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { StaticQuery, graphql } from 'gatsby'
+
+import Header from './header'
+import './layout.css'
+
+const Layout = ({ children }) => (
+  <StaticQuery
```

Comment:
better use here the new `useStaticQuery`? -> https://www.gatsbyjs.org/docs/use-static-query/

Reply:
The example was bootstrapped from a default starter, so we're just using what is the default there.